core-java-modules/core-java-jpms/decoupling-pattern1/consumermodule/pom.xml

```diff
@@ -17,7 +17,7 @@
     <dependencies>
         <dependency>
             <groupId>com.baeldung.servicemodule</groupId>
-            <artifactId>servicemodule1</artifactId>
+            <artifactId>servicemodule</artifactId>
```

Comment:
should this be consumermodule?

Reply:
@lor6 - the artifactId for the module is actually consumermodule, line 20 is adding a dependency on another module, hence it is servicemodule.